package.json

```diff
@@ -39,6 +39,7 @@
     "@trussworks/react-uswds": "^9.1.0",
     "@uswds/uswds": "^3.8.1",
     "autoprefixer": "^10.4.19",
+    "date-fns": "^4.1.0",
```

Comment:
is this necessary?

Reply:
Possibly not. I was hitting errors about that library being missing so I added it, but that might have been related to my `yarn serve` troubles. I'll try removing it and re-testing.

Reply:
Yah, all the dependencies should be supplied by veda ui so this should not be necessary!